SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -653,7 +653,7 @@ void USpatialReceiver::DestroyActor(AActor* Actor, Worker_EntityId EntityId)
 
 void USpatialReceiver::CleanupDeletedEntity(Worker_EntityId EntityId)
 {
-	Cast<USpatialPackageMapClient>(NetDriver->GetSpatialOSNetConnection()->PackageMap)->RemoveEntityActor(EntityId);
+	PackageMap->RemoveEntityActor(EntityId);
```

Comment:
This was causing a crash when shutting down a non-PIE server because the NetConnection is removed from the `ClientConnections` before the channels are cleaned up.